contracts/passport/PassportRegistry.sol

```diff
@@ -108,30 +127,68 @@ contract PassportRegistry is Ownable, Pausable {
     }
 
     /**
-     * @notice Transfer the passport id of the msg.sender to the newWallet.
-     * @dev Can only be called by the passport owner.
+     * @notice Transfers the passport ID of the msg.sender to the new wallet.
+     * @dev Can only be called by the passport owner and when the contract is not paused.
+     * @param newWallet The address of the new wallet to transfer the passport to.
      */
     function transfer(address newWallet) public whenNotPaused {
         uint256 id = passportId[msg.sender];
-        uint256 newWalletId = passportId[newWallet];
+        require(newWallet != msg.sender, "You can not transfer to yourself");
```

Comment:
Should we add a check for the 0 address?

Reply:
sure, we can